src/main/31-cloudfront.tf

```diff
@@ -53,21 +61,20 @@ module "fe_cdn" {
     target_origin_id       = "fe_hosting_oac"
     viewer_protocol_policy = "redirect-to-https"
 
-    allowed_methods = ["GET", "HEAD", "OPTIONS"]
-    cached_methods  = ["GET", "HEAD"]
-    compress        = true
-    query_string    = true
+    allowed_methods          = ["GET", "HEAD", "OPTIONS"]
+    cached_methods           = ["GET", "HEAD"]
+    origin_request_policy_id = data.aws_cloudfront_origin_request_policy.all_viewer.id
+    compress                 = true
   }
   ordered_cache_behavior = [
     {
-      path_pattern           = "/*"
+      path_pattern           = "/eservices/*"
```

Comment:
Just a note: there is a quota limit on how many cache behaviours you can have in a distribution (25).
This looks ok to me, but check with the developers how many path patterns they expect.